### ISSUE TITLE
Hiding the stage but not the sprite pane

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6696]
### Why this would be helpful

> The view proposed by the "Hide stage and sprite pane" addon is great, especially combined with the "Auto-hiding block palette" addon. But, when you have to change often the sprite you are working on, you have to unhide the sprite pane, click the wanted sprite, and hide the sprite pane again. Doing it many times is annoying.
>
> I propose to add an option to the "Hide stage and sprite pane" addon, where you can choose if only the stage is hidden, or both. When only the stage is hidden, the sprite thumbnails are shown in a single column.

_Originally posted by @sfederici in https://github.com/ScratchAddons/ScratchAddons/discussions/5218_

### My suggestion

Make it so that you can still see the sprite pane while the stage is hidden through the `hide-stage` addon.

Something like this:
![Sprite pane with stage hidden](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/7c6670db-c082-4b76-b82a-0ee66f25aa57)

Perhaps this would be good as an option.

**Comment by sfederici** [comment]
> Make it so that you can still see the sprite pane while the stage is hidden through the `hide-stage` addon.
> Something like this: ![Sprite pane with stage hidden](https://user-images.githubusercontent.com/106490990/270192735-7c6670db-c082-4b76-b82a-0ee66f25aa57.png)

I was asking for just one column of sprites as the two columns spritelist would leave the same amount of space for the code area as in the small stage layout, wouldn't it? Instead a single column of sprites would make the code area larger.

A further idea would be moving the stage icon on top of the spritelist, so to leave even more free space.

**Comment by sfederici** [comment]
This can be certainly a useful layout when you have enough space in the coding area but you have so many sprites in the project that looking for them in the spritelist is not easy.

**Comment by WorldLanguages** [comment]
It looks like it would be useful to make the number of columns customizable.

**Comment by DNin01** [comment]
> > Make it so that you can still see the sprite pane while the stage is hidden through the `hide-stage` addon.
> 
> I was asking for just one column of sprites as the two columns spritelist would leave the same amount of space for the code area as in the small stage layout, wouldn't it? Instead, a single column of sprites would make the code area larger.

Yeah! I had that in mind but two columns was easier to mock up at the time.

**Comment by DNin01** [comment]
> It looks like it would be useful to make the number of columns customizable.

That's interesting too.

**Comment by DNin01** [comment]
> ![image](https://github.com/ScratchAddons/ScratchAddons/assets/61996651/266fbb03-9c8e-4b6e-8e9a-6348f8a4523f)

If we go with a design like this, I think it might be better to have a collapse/expand arrow button to hide/show the stage instead of a separate set of switch buttons. The existing buttons for stage size control could then be used to change the sprite pane width.
